# HG changeset patch
# User Fabian Groffen <grobian@gentoo.org>
# Date 1464576141 -3600
#      Mon May 30 03:42:21 2016 +0100
# Node ID 684370c57b850b13c7d6c5da32f8d37f6f40886c
# Parent  ad7b60529625aefec240f7deb9a1fe9f58cea0b2
[PATCH] bdb-prefix
berkely db check is completely broken, but mess with it just enough such
that it works for Gentoo Prefix

In particular make it NOT find magic libs from random locations, instead
require to be in given paths.  Add some versions to detect.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff -r ad7b60529625 -r 684370c57b85 configure.ac
--- a/configure.ac	Mon May 30 03:42:06 2016 +0100
+++ b/configure.ac	Mon May 30 03:42:21 2016 +0100
@@ -1082,7 +1082,7 @@ then
     then
         if test x$ac_bdb_prefix = xyes || test x$ac_bdb_prefix = x
         then
-          ac_bdb_prefix="$mutt_cv_prefix /opt/csw/bdb4 /opt /usr/local /usr"
+          ac_bdb_prefix="$mutt_cv_prefix"
         fi
         for d in $ac_bdb_prefix; do
             bdbpfx="$bdbpfx $d"
@@ -1090,7 +1090,7 @@ then
                 bdbpfx="$bdbpfx $d/$v"
             done
         done
-        BDB_VERSIONS="db-4 db4 db-5 db5 db-5.2 db5.2 db52 db-5.1 db5.1 db51 db-5.0 db5.0 db50 db-4.8 db4.8 db48 db-4.7 db4.7 db47 db-4.6 db4.6 db46 db-4.5 db4.5 db45 db-4.4 db4.4 db44 db-4.3 db4.3 db43 db-4.2 db4.2 db42 db-4.1 db4.1 db41 db ''"
+        BDB_VERSIONS="db db-4 db4 db-5 db5 db-5.2 db5.2 db52 db-5.1 db5.1 db51 db-5.0 db5.0 db50 db-4.8 db4.8 db48 db-4.7 db4.7 db47 db-4.6 db4.6 db46 db-4.5 db4.5 db45 db-4.4 db4.4 db44 db-4.3 db4.3 db43 db-4.2 db4.2 db42 db-4.1 db4.1 db41 db ''"
         AC_MSG_CHECKING([for BerkeleyDB > 4.0])
         for d in $bdbpfx; do
             BDB_INCLUDE_DIR=""
